src/runtime/auto-prompt-manager.js

```diff
@@ -452,6 +467,54 @@ export class AutoPromptManager {
     }
   }
 
+  /**
+   * Returns which type of prompt to display based on the type specified,
+   * the viewport width, and whether the disableDesktopMiniprompt experiment
+   * is enabled.
+   *
+   * If the disableDesktopMiniprompt experiment is enabled and the desktop is
```

Comment:
I think this is backwards? The large prompt will be substituted for the miniprompt.